File: smelly/ignored-test/java-junit/TestOneOneIgnoredWithCause.java
import org.junit.Test;
import org.junit.Ignore;
import static org.junit.Assert.assertTrue;

public class ClassStp {
    @Ignore("This is ignored with cause, no failure")
    @Test
    public void TestOneOneIgnoredWithCause() {
        assertTrue(1 == 2);
    }
}

File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/TestOneOneIgnoredWithCause.cs
using Xunit;

namespace DefaultNamespace
{
    public class ClassStp
    {
        [Fact(Skip = "")]
        public void TestOneOneIgnoredWithCause()
        {
            {
                Assert.True(1 == 2, "This is ignored with cause, no failure");
            }
        }
    }
}
